app/assets/javascripts/app/views/publisher/getting_started_view.js

```diff
@@ -19,58 +19,58 @@ app.views.PublisherGettingStarted = Backbone.View.extend({
   // initiate all the popover message boxes
   show: function() {
     this._addPopover(this.el_first_msg, {
-      trigger: 'manual',
+      trigger: "manual",
       offset: 30,
-      id: 'first_message_explain',
-      placement: 'right',
+      id: "first_message_explain",
+      placement: "right",
       html: true,
-      container: 'body'
+      container: "body"
     }, 600);
     this._addPopover(this.el_visibility, {
-      trigger: 'manual',
+      trigger: "manual",
       offset: 10,
-      id: 'message_visibility_explain',
-      placement: 'bottom',
+      id: "message_visibility_explain",
+      placement: "bottom",
       html: true,
-      container: 'body'
+      container: "body"
     }, 1000);
     this._addPopover(this.el_stream, {
-      trigger: 'manual',
+      trigger: "manual",
       offset: -5,
-      id: 'stream_explain',
-      placement: 'left',
+      id: "stream_explain",
+      placement: "left",
       html: true,
-      container: 'body'
+      container: "body"
     }, 1400);
 
     // hide some popovers when a post is created
-    this.$('.button.creation').click(function() {
-      this.el_visibility.popover('hide');
-      this.el_first_msg.popover('hide');
+    this.$("#submit").click(function() {
+      this.el_visibility.popover("hide");
```

Comment:
In fact there are a lot of uses of the el_ prefix across the Backbone.js code. Some examples are as follows:

```
    // init shortcut references to the various elements
    this.el_input = this.$('#status_message_fake_text');
    this.el_hiddenInput = this.$('#status_message_text');
    this.el_wrapper = this.$('#publisher_textarea_wrapper');
    this.el_submit = this.$('input[type=submit], button#submit');
    this.el_preview = this.$('button.post_preview_button');
    this.el_photozone = this.$('#photodropzone');
```
Should all these el_ prefixes be removed?